### PR TITLE
Add hook 'useClipboard' for use on reusable component and easily to use it

### PR DESCRIPTION
**Pull request by thinnakrit**
```
import {useClipboard} from 'react-copy-to-clipboard';

const CopyButton = () => {
  const [isCopy, setClipboard] = useClipboard()
  const onClickButton = () => {
    setClipboard('Copy Text Example')
  }

  return (
    <div>
      {isCopy && <span>Copied!</span>}
      <button onClick={onClickButton}>Submit</button>
    </div>
  )
}
```

setClipboard: PropTypes.func
isCopy: PropTypes.booleen

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
+const {useClipboard} = require('./Hook');
 
+const ClipboardModules = {
+  CopyToClipboard,
+  useClipboard
+};
 
-CopyToClipboard.CopyToClipboard = CopyToClipboard;
-module.exports = CopyToClipboard;
+module.exports = ClipboardModules;
```
Comment (nkbt) [inline src/index.js:9]: This is a breaking change.
Previous implementation allowed both `default` import and `named` import

File: README.md [hunk 2]
```diff
@@ -119,6 +119,30 @@ CopyToClipboard is a simple wrapping component, it does not render any tags, so
...
+    setClipboard('Copy Text Example')
+  }
+
+  return (
+    <div>
+      {isCopy && <span>Copied!</span>}
+      <button onClick={onClickButton}>Submit</button>
+    </div>
+  )
+}
```
Comment (nkbt) [inline README.md:139]: How is this better than using lib directly?

```suggestion
import copy from 'copy-to-clipboard';

const CopyButton = () => {
  const [isCopied, setIsCopied] = useState(false)
  const onClickButton = () => {
    copy('Copy Text Example')
    setIsCopied(true)
  }

  return (
    <div>
      {isCopied && <span>Copied!</span>}
      <button onClick={onClickButton}>Submit</button>
    </div>
  )
}
```

**Review by nkbt — COMMENTED** [review]
I really struggle to convince myself this is necessary (see suggestion for readme example)

File: README.md (hunk 2 above)
Comment (thinnakrit) [reply README.md:139]: Now i use ```import copy from 'copy-to-clipboard'``` directly on my project
but i want to change use it on you lib ;)

also
can use with out lib
```
const copyToClipboard = (text) => {
  if (document && text) {
      var textField = document.createElement('textarea')
      textField.innerText = text
      document.body.appendChild(textField)
      textField.select()
      document.execCommand('copy')
      textField.remove()
  }
}
```

Comment (nkbt) [reply README.md:139]: Yes exactly!

**Pull request closed without merging**